frontend/src/app/main/ui/formats.cljs

```diff
@@ -67,9 +67,11 @@
   (let [sizing (if (= type :width)
                  (:layout-item-h-sizing shape)
                  (:layout-item-v-sizing shape))]
-    (if (= sizing :fill)
-      "100%"
-      (str (format-pixels value)))))
+    (if (string? value)
```

Comment:
Two things, first it needs to consider the `sizing = auto` also I think the size type has priority over the value. 

My sugestion would be:

```clojure
(case sizing
  :fill ...
   :auto ....
   (if (number? value)
       (format-pixels value)
       value))    
```